README.md

```diff
@@ -1,8 +1,8 @@
-<img src="logo_pypiper.svg" alt="Pypiper logo" height="70" align="left"/>
+<img src="https://raw.githubusercontent.com/databio/pypiper/master/logo_pypiper.svg?sanitize=true" alt="Pypiper logo" height="70" align="left"/>
 
 # Pypiper
 
 [![Documentation Status](https://readthedocs.org/projects/pypiper/badge/?version=latest)](http://pypiper.readthedocs.org/en/latest/?badge=latest)
 [![Build Status](https://travis-ci.org/databio/pypiper.svg?branch=master)](https://travis-ci.org/databio/pypiper)
 
-A lightweight python toolkit for gluing together restartable, robust command line pipelines. The best place to learn more is at the [documentation hosted at Read the Docs](http://pypiper.readthedocs.org/).
+A lightweight python toolkit for gluing together restartable, robust command line pipelines. The best place to learn more is at the [documentation](http://code.databio.org/pypiper/).
```

Comment:
How about "creating" or "building" instead of "gluing together." Current wording seems to suggest that this is a repair tool rather than something that can also be used for new constructions.

Reply:
great!